*** testsBoth/A-041-041-M-ok.out	2023-10-19 17:31:45.000000000 +0100
--- testsBoth/A-041-041-M-ok.outhyp	2023-10-26 01:21:18.300780908 +0100
***************
*** 16,23 ****
  7 - Mostrar 'cut buffer'
  0 - Sair
  Escolha uma opção: 2
- Especifique a gama (startline;startcol:endline;endcol): 2;1
- Insira o conteúdo da célula: =1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 16,21 ----
***************
*** 27,32 ****
--- 25,34 ----
  6 - Colar
  7 - Mostrar 'cut buffer'
  0 - Sair
+ Escolha uma opção: 2;1
+ Número inválido!
+ Escolha uma opção: =1;1
+ Número inválido!
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 1;1
  1;1|5
***************
*** 53,59 ****
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;1
! 2;1|5=1;1
  Menu de Edição
  1 - Visualizar
  2 - Inserir
--- 55,61 ----
  0 - Sair
  Escolha uma opção: 1
  Especifique a gama (startline;startcol:endline;endcol): 2;1
! 2;1|25
  Menu de Edição
  1 - Visualizar
  2 - Inserir
