*** testsBoth/A-075-075-M-ok.out	2023-10-19 17:31:47.000000000 +0100
--- testsBoth/A-075-075-M-ok.outhyp	2023-10-26 01:21:21.684777519 +0100
***************
*** 7,13 ****
  0 - Sair
  Escolha uma opção: 1
  Guardar antes de fechar? s
! Ficheiro sem nome. Guardar ficheiro como: saved27
  Especifique o número de linhas da folha: 34
  Especifique o número de colunas da folha: 43
  Menu Principal
--- 7,13 ----
  0 - Sair
  Escolha uma opção: 1
  Guardar antes de fechar? s
! Especifique o número de linhas da folha: saved27
  Especifique o número de linhas da folha: 34
  Especifique o número de colunas da folha: 43
  Menu Principal
